eclass/coreos-go.eclass

```diff
@@ -9,41 +9,52 @@
 # @DESCRIPTION:
 # Name of the Go package unpacked to ${S}, this is required.
 
-[[ ${EAPI} != "5" ]] && die "Only EAPI=5 is supported"
+[[ ${EAPI} != "5" ]] && die "${ECLASS}: Only EAPI=5 is supported"
 
 inherit flag-o-matic multiprocessing toolchain-funcs
 
 DEPEND="dev-lang/go"
 
+# @FUNCTION: go_get_arch
+# @USAGE: export GOARCH=$(go_get_arch)
+go_get_arch() {
```

Comment:
conveniently so far these match portage's arch names, available in $ARCH. Should we just do `GOARCH="${ARCH}"`?